addons/l10n_in/models/l10n_in_pan_entity.py

```diff
@@ -0,0 +1,59 @@
+from stdnum.in_ import pan
+from odoo import api, fields, models
+
+
+class L10nInPanEntity(models.Model):
+    _name = 'l10n_in.pan.entity'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+    _description = 'Indian PAN Entity'
+
+    name = fields.Char(string="Number", tracking=1)
```

Comment:
I suppose the PAN entity is shared among companies (multi-company environment), but I suppose the TDS deduction is the same then for all companies?